data.js

```diff
@@ -1,51 +1,52 @@
-import { inputFocus } from "./script.js";
+import { inputFocus, resultDataUl } from "./script.js";
 console.log("data");
 //*FETCH THE BREED DATA FROM THE API
-export const fetchBreedData = async (
-  resultDataUl,
-  notification,
-  searchInput
-) => {
+export const fetchBreedData = async (resultDataUl, notification) => {
   try {
     const resolve = await axios.get(`https://api.thedogapi.com/v1/breeds/`);
     return resolve.data;
   } catch (error) {
-    catchError(resultDataUl, notification, searchInput);
+    return catchError(resultDataUl, notification);
   }
 };
 
-export const catchError = function (resultDataUl, notification, searchInput) {
+export const catchError = function (resultDataUl, notification) {
   resultDataUl.innerHTML = "";
   notification = document.createElement("li");
   notification.classList.add("notification");
   notification.textContent =
     " Not found. Something went wrong. Please try again later";
   resultDataUl.append(notification);
-  inputFocus(searchInput);
+  inputFocus();
 };
 
 //*CREATE AND DISPLAY li WITH THE FETCH DATA FROM resultFetchBreed
-export const showAllData = function (resultFetchBreed, resultDataUl, newLi) {
+export const showAllData = function (resultFetchBreed, resultDataUl) {
   resultDataUl.innerHTML = "";
   if (resultFetchBreed === undefined) {
```

Comment:
v katerem primeru je lahko `resultFetchBreed` undefined?

Reply:
Takrat ko je npr nepravilen url,  in mi vrže prazno dato oziroma catchError funkcijo, vendar, pa če ne dam tega `if (resultFetchBreed === undefined) {` mi pa vrže error `Uncaught TypeError: Cannot read properties of undefined (reading 'length')`, ker potem v `addDataToHtml `fn uporabi točno to `resultFetchBreed ` dato in loopa čez in če je prazna napiše
ta error

Reply:
glih to.. če je na api requestu error pol se ne bi niti smela `showAllData` fn klicat.. a ni tko?

pol se kliče `catchError` in je.. no ja, lahko to v naslednih PRjih popraviš 👍